packages/react-scripts/template/src/registerServiceWorker.js

```diff
@@ -24,12 +50,12 @@ export default function register() {
                   // the fresh content will have been added to the cache.
                   // It's the perfect time to display a "New content is
                   // available; please refresh." message in your web app.
-                  console.log('New content is available; please refresh.');
+                  showMessage('New content is available. Please refresh.');
                 } else {
                   // At this point, everything has been precached.
                   // It's the perfect time to display a
                   // "Content is cached for offline use." message.
-                  console.log('Content is cached for offline use.');
+                  showMessage('Content is cached for offline use.');
```

Comment:
Since Service Workers are not enabled in development mode, maybe we should let the developer know why the toast exists. 

We could also add some information in `registerServiceWorker.js` about how to remove service workers.

> "In production your users will see this toast to notify them [service workers](https://github.com/facebookincubator/create-react-app/tree/master/packages/react-scripts/template#making-a-progressive-web-app) are enabled. See registerServiceWorker.js for more info"

What about instead of using the word 'content' we say 'app'.
> A new version of this app is available. Please refresh your browser.

> This app had been made available for offline use.